src/utils/sanitizeFilename.js

```diff
@@ -0,0 +1,5 @@
+const sanitizeFilename = require('sanitize-filename');
+
+module.exports = function sanitize(name, replacement = '-') {
+  return sanitizeFilename(unsafeRegex, {replacement});
```

Comment:
Is this un-tested?

`unsafeRegex` is no longer defined.
`name` is not passed to `sanitizeFilename`.

Doesn't look like this code would work as-is. The helper is probably unnecessary now, too. Can just use `sanitize-filename` directly.

Reply:
oh shoot, it was unpushed ! i'd amended the commit, and had not forced push.

i thought it preferable to keep the indirection in case more was ever desired of sanitize filename than simple replacement (such as removing the namespace, or simply removing all non alpha-numerics). but fair enough. will do.